packages/dds/tree/src/simple-tree/proxies.ts

```diff
@@ -44,6 +44,8 @@ import {
 import { cursorFromFieldData, cursorFromNodeData } from "./toMapTree.js";
 import { IterableTreeArrayContent, TreeArrayNode } from "./treeArrayNode.js";
 import { TreeNode, Unhydrated } from "./types.js";
+// eslint-disable-next-line import/no-internal-modules
+import { LazyIdentifierField } from "../feature-libraries/flex-tree/lazyField.js";
 
```

Comment:
@noencke I am currently using this to check that a field is of type identifier, so that I could use it's uuid method to return the stable id. I did not export this field in our index.ts files, as none of the other LazyFields are exported. Would there be a better way for me to make this check so I can call the uuid method on LazyIdentifierField in this file?

Reply:
It's okay for now to have an import like this. We have a couple of other dependencies like this already, where the proxies assume some stuff about the flex tree.

Alternatively, you could just do the conversion in this layer. You can get context.nodeKeys off of the field. You should be able to do weaker casts too. Right now you have:

```ts
class LazyIdentifierField<TTypes extends FlexAllowedTypes>
	extends LazyValueFieldReadOnly<TTypes>
	implements FlexTreeRequiredField<TTypes>
```

but identifier flex fields should be hard-coded to number types. Something like this:

```ts
class LazyIdentifierField
	extends LazyValueFieldReadOnly<[LeafNodeSchema<string, ValueSchema.Number>]>
	implements FlexTreeRequiredField<[LeafNodeSchema<string, ValueSchema.Number>]>
```

Then when you do `field.content`, you will get a `number`, so you don't need to cast to `unknown`.

Reply:
Updated `schemaFactory.identifier` to have assert for type string.